### ISSUE TITLE
Call-time pass-by-reference false positive

### ISSUE THREAD

**Issue by bwhitty** [squizlabs/PHP_CodeSniffer#672]
Running `phpcs` as follows 

```phpcs --standard=Generic --extensions=php --sniffs=Generic.Functions.CallTimePassByReference -pvs ./```

throws an error in the phpseclib file found here: https://github.com/phpseclib/phpseclib/blob/0.3.8/phpseclib/File/X509.php#L4382

This does not fail a PHP lint check when running 5.4+.

**Comment by gsherwood** [comment]
Thanks for reporting.
